content/en/docs/getting-started/_index.md

```diff
@@ -60,73 +61,141 @@ olm-operator       1/1     1            1           5m52s
 packageserver      2/2     2            2           5m43s
 ```
 
-## Installing an Operator using OLM
 
-When you install OLM, it comes packaged with a number of Operators developed by the community that you can install instantly.
-You can use the `pacakagemanifest` api to see the operators available for you to install in your cluster:
+## Installing OLM and operator bundle via SDK for development purpose
 
-```sh
-$ kubectl get packagemanifest -n olm
-NAME                               CATALOG               AGE
-cassandra-operator                 Community Operators   26m
-etcd                               Community Operators   26m
-postgres-operator                  Community Operators   26m
-prometheus                         Community Operators   26m
-wildfly                            Community Operators   26m
+### Installing OLM
+
+The [`operator-sdk`][operator-sdk] provides a command to easily install and uninstall OLM for development purposes. See the [SDK installation guide][sdk-installation-guide] on how to install `operator-sdk`.  
+
+With `operator-sdk` installed, you can easily install OLM on your cluster by running `operator-sdk olm install`. It is just as easy to uninstall OLM by running `operator-sdk uninstall olm`.  For more, information about how to integrate your project using [`operator-sdk`][operator-sdk] CLI tool see the [OLM integration][sdk-olm-integration] section. 
+
+### Installing an operator bundle
+
+You can use the [`operator-sdk`][operator-sdk] CLI to run your bundle with [`operator-sdk run bundle`](https://sdk.operatorframework.io/docs/cli/operator-sdk_run_bundle/).
+
+## Running OLM locally with minikube for development purpose
+
+This command starts minikube, builds the OLM containers locally with the minikube-provided docker, and uses the local configuration in [local-values.yaml][local-values.yaml] to build localized deployment resources for OLM.
+
+```bash
+# To install and run locally
+$ make run-local
```

Comment:
We'll want to remove the `$` to make it easy to copy-and-paste or else the linter will complain as there's no follow command output.